File: joaca-de-a-invatatul/jsp-tests/src/main/java/org/example/web/domain/Asset.java
package org.example.web.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Asset {
  private int id;
  private int userId;
  private String name;
  private String description;
  private int value;
}

File: joaca-de-a-invatatul/AspMVCex/AspMVCex/Models/Asset.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AspMVCex.Models
{
    public class Asset
    {
        public int id { get; set; }

        public int userId { get; set; }

        public int value { get; set; }


        public string name { get; set; }
        public string description { get; set; }
    }
}